package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
When you are creating package.json file you can use `npm init` command: https://docs.npmjs.com/cli/v10/commands/npm-init

It will ask you several questions and give you minimum valid package.json configuration

Or event `npm init -y`, it will apply all default values